src/v2/mod.rs

```diff
@@ -745,6 +747,40 @@ impl Client {
         });
         Ok(stream)
     }
+
+    pub async fn peer_connect(&mut self, addr: SocketAddr) -> endpoints::RPCResult<bool> {
```

Comment:
Similar to the comments I had in the grpc-api respository, I don't see the `bool` adding value.